contracts/payloads/IGP85/PayloadIGP85.sol

```diff
@@ -0,0 +1,548 @@
+
+        address REWARDS_ADDRESS = address(
+            //0x
+        );
+
+        {
+            /// fUSDC
+            IFTokenAdmin(F_USDC).updateRewards(REWARDS_ADDRESS);
+
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDC
+            );
+
+            protocols[0] = F_USDC;
+            tokens[0] = USDC_ADDRESS;
+            amounts[0] = allowance ;
+        }
+
+        {
+            /// fUSDT
+            IFTokenAdmin(F_USDT).updateRewards(REWARDS_ADDRESS);
+
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDT
+            );
+
+            protocols[1] = F_USDT;
+            tokens[1] = USDT_ADDRESS;
+            amounts[1] = allowance ;
+        }
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+
+        {
+            ILendingRewards(REWARDS_ADDRESS).start();
+        }
+    }
+
+    // @notice Action 5: Constrict BOLD DEX
+    function action5(){
+
+        {
+            address USDC_BOLD_DEX = getDexAddress(25);
+            // USDC-BOLD DEX
+            {
+                // USDC-BOLD Dex
+                Dex memory DEX_USDC_BOLD = Dex({
+                    dex: USDC_BOLD_DEX,
+                    tokenA: USDC_ADDRESS,
+                    tokenB: BOLD_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10, // $10
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_USDC_BOLD); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(USDC_BOLD_DEX, TEAM_MULTISIG, false);
+            }
+        }
+
+        {
+            address USDC_BOLD_fSL20 = 0x9b1f75ea07723F331996831f6d04AD4900d1A3B3;
+
+            // Update USDC-BOLD smart lending vault supply limit with minimum value
+            IFluidAdminDex.UserSupplyConfig[]
+                memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+                config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: USDC_BOLD_fSL20,
+                expandPercent: 1 * 1e2, // 1%
+                expandDuration: 999 days, // max time
+                baseWithdrawalLimit: 1 * 1e18 // 1 shares
+            });
+
+            IFluidDex(USDC_BOLD_fSL20).updateUserSupplyConfigs(config_);
+        }
+
+        {
+            // Pause user supply and borrow
+            address[] memory supplyTokens = new address[](2);
+            supplyTokens[0] = USDC_ADDRESS;
+            supplyTokens[1] = BOLD_ADDRESS;
+
+            address[] memory borrowTokens = new address[](2);
+            borrowTokens[0] = address(0);
+            borrowTokens[1] = address(0);;
+
+            // Pause the user operations
+            pauseUser(USDC_BOLD_DEX, supplyTokens, borrowTokens);
+        }
```

Comment:
let's also do this here on the USDC_BOLD_DEX:

maxSupplyShares(1), max borrowShares(1)